src/elements/Progressbar/style.scss

```diff
@@ -0,0 +1,22 @@
+:local {
+  @import '../../styles/config';
+  $progress-bar-height: 13px;
+  $bar-background-color: #0290d7;
+  $container-background-color: #e6e6e9;
```

Comment:
please use predefined gray color in config.scss to make our components are consistent.